code/modules/mining/coins.dm

```diff
@@ -44,6 +44,28 @@
 	name = "platinum coin"
 	icon_state = "coin_adamantine"
 
+/obj/item/weapon/coin/erp
+	name = "erp token"
```

Comment:
ERP needs to be capitalized. It is the proper term for the unit of currency. Lower case is colloquial.